ui/riff-raff.yaml

```diff
@@ -2,11 +2,17 @@ stacks: [frontend]
 regions: [eu-west-1]
 
 deployments:
+  cloudformation:
+    type: cloud-formation
+    parameters:
+      cloudFormationStackName: automat-ui
+      templatePath: cfn.yaml
+      cloudFormationStackByTags: false
   s3:
     type: aws-s3
-    app: automat-ui
+    dependencies: [cloudformation]
     parameters:
-      bucket: aws-frontend-automat-ui
+      bucket: com-gu-automat-ui
```

Comment:
We have buckets prefixed `aws-frontend`, `frontend`, and `com-gu`, but I like the latter as bucket names are global and using domains seems more robust.